docs/source/deployment/databricks/databricks_ide_development_workflow.md

```diff
@@ -10,263 +16,91 @@ By working in your local environment, you can take advantage of features within
-
- ```{note}
-If you are not using the `databricks-iris` starter to create a Kedro project, **and** you are working with a version of Kedro **earlier than 0.19.0**, then you should [disable file-based logging](https://docs.kedro.org/en/0.18.14/logging/logging.html#disable-file-based-logging) to prevent Kedro from attempting to write to the read-only file system.
- ```
-
-### Create a Repo on Databricks
-
-Create a new Repo on Databricks by navigating to `New` tab in the Databricks workspace UI side bar and clicking `Repo` in the drop-down menu that appears.
-
-In this guide, you will not sync your project with a remote Git provider, so uncheck `Create repo by cloning a Git repository` and enter `iris-databricks` as the name of your new repository:
-
-![Create a new Repo on Databricks](../../meta/images/databricks_repo_creation.png)
+### Create a new Kedro Project
 
-### Sync code with your Databricks Repo using dbx
+## Create the Databricks Asset Bundles using `kedro-databricks`
 
-The next step is to use dbx to sync your project to your Repo.
+`kedro-databricks` is a wrapper around the `databricks` CLI. It's the simplest way to get started without getting stuck with configuration. To learn more about Databricks Asset Bundles and customization, read [What are Databricks Asset Bundles](https://docs.databricks.com/en/dev-tools/bundles/index.html).
 
-**Open a new terminal instance**, activate your conda environment, and navigate to your project directory and start `dbx sync`:
+1. Install `kedro-databricks`:
 
 ```bash
-conda activate iris-databricks
-cd <project_root>
-dbx sync repo --dest-repo iris-databricks --source .
+pip install kedro-databricks
 ```
 
-This command will sync your local directory (`--source .`) with your Repo (`--dest-repo iris-databricks`) on Databricks. When started for the first time, `dbx sync` will write output similar to the following to your terminal:
+2. Initialize the Databricks configuration:
 
 ```bash
-...
-[dbx][2023-04-13 21:59:48.148] Putting /Repos/<databricks_username>/iris-databricks/src/tests/__init__.py
-[dbx][2023-04-13 21:59:48.168] Putting /Repos/<databricks_username>/iris-databricks/src/tests/test_pipeline.py
-[dbx][2023-04-13 21:59:48.189] Putting /Repos/<databricks_username>/iris-databricks/src/tests/test_run.py
-[dbx][2023-04-13 21:59:48.928] Done. Watching for changes...
-```
-
-**Keep the second terminal (running dbx sync) alive during development; closing it stops syncing new changes.**
-
-`dbx sync` will automatically sync any further changes made in your local project directory with your Databricks Repo while it runs.
-
-```{note}
-Syncing with dbx is one-way only, meaning changes you make using the Databricks Repos code editor will not be reflected in your local environment. Only make changes to your project in your local environment while syncing, not in the editor that Databricks Repos provides.
+kedro databricks init
```

Comment:
I must be missing some authentication step because I'm getting an error right here:

```
Please select your cloud provider:
1. Azure
2. AWS
3. GCP
 [1]: 1
Creating a Databricks asset bundle definition...
Error: failed to compute file content for databricks.yml.tmpl. template: :43:15: executing "" at <is_service_principal>: error calling is_service_principal: unexpected error handling request: json: cannot unmarshal number into Go struct field APIErrorBody.error_code of type string. This is likely a bug in the Databricks SDK for Go or the underlying REST API. Please report this issue with the following debugging information to the SDK issue tracker at https://github.com/databricks/databricks-sdk-go/issues. Request log:
```
GET /api/2.0/preview/scim/v2/Me
> * Host: 
> * Accept: application/json
> * Authorization: REDACTED
> * Traceparent: 00-060ff8dbd320810343acb3b6ffe77db8-5cfa9826a56a1358-01
> * User-Agent: cli/0.223.1 databricks-sdk-go/0.43.0 go/1.21.11 os/darwin cmd/bundle_init auth/pat
< HTTP/2.0 403 Forbidden
< * Content-Length: 52
< * Content-Type: application/json; charset=utf-8
< * Date: Mon, 11 Nov 2024 15:58:48 GMT
< * Server: databricks
< * Vary: Accept-Encoding
< * X-Databricks-Reason-Phrase: Invalid access token.
< * X-Request-Id: f5733af1-fa9d-49fa-bcc5-b93b0ddb79d9
< {
<   "error_code": 403,
<   "message": "Invalid access token."
< }
```

Reply:
Indeed, my credentials were outdated (and so was my databricks CLI).

Maybe we should add a `databricks configure` or `databricks auth login` step? Or at least a command that checks whether proper auth is in place, like `databricks auth profiles`? (Not sure if there's a better command to do this)

Reply:
Indeed, the section above that you find empty includes setting up the databricks CLI first.